dcos-log/api/v1/handlers.go

```diff
@@ -17,6 +17,15 @@ import (
 	"github.com/gorilla/mux"
 )
 
+type key int
+
+var streamKey key = 1
+
+func requestStreamKeyFromContext(ctx context.Context) bool {
```

Comment:
no, this one is coming from https://github.com/dcos/dcos-log/pull/40/files#diff-57564f1e160250bafe5d17a31d188f28L12